src/TestableIO.System.IO.Abstractions.TestingHelpers/IMockFileDataAccessor.cs

```diff
@@ -110,4 +111,9 @@ public interface IMockFileDataAccessor : IFileSystem
     /// Gets a reference to the underlying file system. 
     /// </summary>
     IFileSystem FileSystem { get; }
+    
+    /// <summary>
+    /// Gets the event system for this file system.
+    /// </summary>
+    MockFileSystemEvents Events { get; }
```

Comment:
Why do you need to add it here also? You already can get access to the events via `FileSystem.Events`.